sdk/tests/iot/adu/test_az_iot_adu.c

```diff
@@ -29,14 +29,16 @@
 
 static uint8_t expected_agent_state_payload[]
```

Comment:
Do we need to update the manifest version in these tests as well for this change?

Reply:
Good question. We can. The tests are just parsing out a number so it realistically could be `10000` and it wouldn't matter. As long as the format of the payload is correct.